models/studentDataModel.js

```diff
@@ -0,0 +1,37 @@
+/* dataModel.js
+*/
+​
+var mongoose = require('mongoose');   
+​
+var dataSchema = mongoose.Schema({
+	type: String,
+	date: Date,
+	data: Array
+});  
+​
+var studentSchema = mongoose.Schema({
+	id: Number,
+	name: String,
+	program: String,
```

Comment:
add
`archived: {type: Boolean, required: false}`
so it's true or doesn't exist (or just false, but would just make adding new students a little bit simpler)